src/main/tut/index.html

```diff
@@ -126,6 +132,9 @@
 printInt
 ```
 
+<!-- COMMENT: It *might* be worth mentioning context bounds here, as it would cleanup some of the later code examples -->
```

Comment:
Thought about it, but I'm not sure - there's already quite a bit of syntax to go through (a lot of people are not aware that you can explicitly specify type parameters, for example) and I worry that this might be too much.

To be perfectly honest, I was thinking of a _second_ presentation about best practices, such as common companion object methods (instance creation and summoning), operators, context bounds, default instances... and maybe a few of the common tools (imp, machinist, simulacrum, typeclassic is that ever sees the day...)